src/resources/SearchAnalysis/SearchAnalysis.ts

```diff
@@ -0,0 +1,15 @@
+import dayjs from 'dayjs';
+import API from '../../APICore.js';
+import Resource from '../Resource.js';
+import {ReplayAnalysis} from './SearchAnalysisInterface.js';
+
+export default class SearchAnalysis extends Resource {
+    static baseUrl = '/rest/search/v3/analysis';
+
+    replay(id: string, from = dayjs().subtract(1, 'week').format('YYYY-MM-DD')) {
```

Comment:
Is there a way to provide the inclusive date at which to stop looking for the SearchUID (`to`)